src/components/com_tjcertificate/media/js/certificateImage.js

```diff
@@ -82,5 +82,21 @@ var certificateImage = {
 		setTimeout(function() {
 			jQuery("#copyurl").popover("hide");
 		}, 1000);
-	}
+	},
+
+    copySingleUrl: function(element) {

```

Comment:
Well here is now the code duplication, use only one function and make it work for both front-end and backend @divyachaudhari